src/main/kotlin/com/github/huntersherms/pojo2parquet/Pojo2Parquet.kt

```diff
@@ -114,7 +115,7 @@ class Pojo2Parquet<T>(private val clazz: Class<T>) {
      * Use in place of parquet2Pojos if your POJOs are Jackson annotated and you want Jackson to manage the mapping
      * to your POJOs.
      */
-    fun parquet2JacksonAnnotatedPojos(file: File): List<T> {
+    fun parquet2JacksonAnnotatedPojos(file: File, mapper: AvroMapper = getAvroMapper()): List<T> {
```

Comment:
should this return `Collection<T>` instead of `List` ? since you generic `Collection<T>` when you do `jacksonAnnotatedPojos2Parquet()`.

Reply:
No, because this implementation will specifically return a List<T>, while the pojos2Parquet methods can consume any Collection<T>

Reply:
Good question... Do we want to enable future flexibility (we may decide to use a different underlying implementation) or do we want to be explicit about the type? I'll think on that.

(Technically List is the Interface and ArrayList is the underlying implementation)

Reply:
My preference would be to leave it as List to avoid the need to cast on the calling side.